src/scripts/background.js

```diff
@@ -246,11 +246,11 @@ window.TogglButton = {
               TogglButton.setCanSeeBillable();
               ga.reportOs();
             } else {
-              bugsnagClient.notify(new Error(`Fetch user failed ${xhr.status}`), {
-                metaData: {
+              bugsnagClient.notify(new Error(`Fetch user failed ${xhr.status}`), evt => {
```

Comment:
According to `UPGRADING.md` we can just use static here `import Bugsnag from '@bugsnag/js'` but I don't really like importing some magic singleton :shrug: 
So if it still works then I'm happy with it.

Reply:
The upgrading docs [also state](https://github.com/bugsnag/bugsnag-js/blob/master/UPGRADING.md#new-static-interface) that:
> You can choose to hold on to the Client returned by Bugsnag.start(), or not. After ensuring Bugsnag.start() is called first, you can call any Client method on the static interface, which forwards the method call onto the initialized client

Since we're also returning some other object from our wrapper library if we can't initialize it, I though it best to still use the returned client 👍🏻